clippy.toml

```diff
@@ -1,3 +1,7 @@
 avoid-breaking-exported-api = false
+
```

Comment:
here is still an unwrap:
https://github.com/geo-engine/geoengine/blob/a17c117c875a83d3566b56efc977ce31c35e6401/datatypes/src/lib.rs#L60

so I guess macros are not checked?

Reply:
It looks like it:
[a38e04b](https://github.com/geo-engine/geoengine/pull/924/commits/a38e04b63310ff99cf9fe9080b65a045a518b4df)

But it could also be since the macro code is expanded only in test modules, that it was no problem.